src/components/Editable.tsx

```diff
@@ -473,15 +473,17 @@ const Editable = ({ disabled, isEditing, simplePath, path, cursorOffset, showCon
 
   /** Flushes edits and updates certain state variables on blur. */
   const onBlur = () => {
+
+    dispatch(setEditingValue(null))
+
     if (isMobile && isSafari()) {
       resetToolbarPosition()
       document.removeEventListener('scroll', updateToolbarPositionOnScroll)
     }
 
     const { invalidState } = state
     throttledChangeRef.current.flush()
-    // set editingValue in order to position superscript correctly if edited thought is duplicate
-    oldValueRef.current && dispatch(setEditingValue(oldValueRef.current))
```

Comment:
Superscript positioning is broken anyway and needs to be solved a different way, so we can remove this. See #894.